tests/unit/api/bucketPutACL.js

```diff
@@ -301,6 +301,104 @@ describe('putBucketACL API', () => {
         });
     });
 
+    it('should set ACLs with an empty AccessControlList section', done => {
+        const testACLRequest = {
+            bucketName,
+            namespace,
+            headers: { host: `${bucketName}.s3.amazonaws.com` },
+            post: '<AccessControlPolicy xmlns=' +
+                    '"http://s3.amazonaws.com/doc/2006-03-01/">' +
+                  '<Owner>' +
+                    '<ID>852b113e7a2f25102679df27bb0ae12b3f85be6' +
+                    'BucketOwnerCanonicalUserID</ID>' +
+                    '<DisplayName>OwnerDisplayName</DisplayName>' +
+                  '</Owner>' +
+                  '<AccessControlList></AccessControlList>' +
+                '</AccessControlPolicy>',
+            url: '/?acl',
+            query: { acl: '' },
+        };
+
+        bucketPutACL(authInfo, testACLRequest, log, err => {
+            assert.strictEqual(err, undefined);
```

Comment:
I think you have mentioned that AWS doesn't accept empty or multiple ACLs but it doesn't seem to be what being tested here.